package.json

```diff
@@ -97,32 +98,32 @@
     "markdown-link-check": "3.11.2",
     "mocha": "10.2.0",
     "named-html-entities-json": "1.0.0",
-    "prettier": "3.0.3",
     "svg-path-bbox": "1.2.5",
     "svg-path-segments": "1.0.0",
     "svglint": "2.4.0",
     "svgo": "3.0.2",
     "svgpath": "2.6.0",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "xo": "0.58.0"
   },
   "scripts": {
-    "build": "node scripts/build/package.js",
-    "clean": "node scripts/build/clean.js",
-    "format": "prettier --cache --write .",
+    "build": "./scripts/build/package.js",
+    "clean": "./scripts/build/clean.js",
+    "format": "prettier --cache --write --ignore-unknown '**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts|svg)' && xo --fix",
     "lint": "npm run ourlint && npm run jslint && npm run jsonlint && npm run svglint && npm run wslint",
-    "ourlint": "node scripts/lint/ourlint.js",
-    "jslint": "prettier --cache --check .",
-    "jsonlint": "node scripts/lint/jsonlint.js",
-    "svglint": "svglint --ci $npm_config_icons",
```

Comment:
We need to add a  `prettierlint` script to run Prettier with `--check` in the `lint` script with the rest of the parameters of the `format` script.